sos/report/plugins/hbn.py

```diff
@@ -12,13 +12,30 @@
 
 class HBN(Plugin, IndependentPlugin):
 
-    short_desc = 'HBN (Host Based Network)'
+    short_desc = 'HBN (Host Based Networking)'
     plugin_name = "hbn"
+    vsctl = "ovs-vsctl"
     packages = ('hbn-repo',)
 
     def setup(self):
         self.add_copy_spec([
             '/etc/mellanox',
             '/var/log/doca/hbn',
             '/var/lib/hbn/etc/cumulus',
+            '/var/log/containers',
+            self.path_join('/var/log', 'sfc-install.log'),
```

Comment:
why path join is needed? isn't the path fixed?

Reply:
I only wanted specific files and not everything in this folder, hence

Reply:
is that collected by default when running the tool?

Reply:
Removed the path_join as copy_spec can be used directly.
Believed path_join was the way by https://github.com/hareeshpc/doca-sosreport/blob/main/sos/report/plugins/openvswitch.py#L64